prac_07/guitar.py

```diff
@@ -1,6 +1,6 @@
 """
 CP1404 Practical
-Classes for guitars.py program
+Classes for guitars.py program.
 """
 
 
```

Comment:
In your __init__:
```
def __init__(self, name="", year=0, cost=0):
```

I would personally adjust the cost to be a float, as currently it looks like it should be initialized as an Integer.  For example

```
def __init__(self, name="", year=0, cost=0.0):
```